welcome-message/banner.html

```diff
@@ -3,8 +3,8 @@ <h6 class="n-welcome-banner__heading n-welcome-banner__heading--minimized n-welc
 	<div class="o-expander__content o-expander__content--expanded n-welcome-banner__column n-welcome-banner__column--fill" aria-hidden="false">
 		<h6 class="n-welcome-banner__heading n-welcome-banner__column n-welcome-banner__column--m">Welcome to the new FT.com.</h6>
 		<p class="n-welcome-banner__column n-welcome-banner__column--xl n-welcome-banner__strapline">The same global insight. Faster than ever before on all your devices.</p>
-		<p class="n-welcome-banner__column n-welcome-banner__column--fill">
-			<a href="/tour" class="n-welcome-banner__button" data-trackable="tour-page" data-component="cta-take-tour"><span class="n-welcome-banner__cta--s">Take a </span>tour</a>
+		<p class="n-welcome-banner__column">
+			<a href="/tour" class="n-welcome-banner__button" data-trackable="tour-page" data-component="cta-take-tour"><span class="n-welcome-banner__cta--s">Take a </span>Tour</a>
```

Comment:
How come it‘s changing to `Tour`? It should either be `a tour` or `the Tour`, no?